src/ucp/tag/tag_match.h

```diff
@@ -44,9 +44,14 @@ typedef struct ucp_tag_match {
     size_t                    thresh;         /* All messages above the threshold are posted
                                                  to the transport. Either UCP user buffer
                                                  itself or preregistered internal UCP buffer
-                                                 is posted. */
+                                                 is posted. Messages below the threshold are
+                                                 not posted and will be matched in SW. */
     size_t                    zcopy_thresh;   /* All UCP user buffers above the threshold
-                                                 are posted to the transport. */
+                                                 are posted to the transport. For buffers
```

Comment:
Okay...I'm trying to translate this. @yosefe - is this correct description ?

For tresh:
```
Receive message buffers allocated by user or using UCP bouncing buffer with the message size above the threshold are posted to the transport's receive tag matching queue (hardware).
```
For zcopy_thresh
``` Receive messages buffers allocated by user with the message size above the threshold are directly posted to the transport's receive tag matching queue (hardware).  Once matching message appears it is directly placed into the buffer.
```